### PR TITLE
CALS-5426 Dropdown Chevron from Income Interval Covers up the text in…

### PR DESCRIPTION
**Pull request by vladimirPe**
gridClassName was changed

#### Which User story does this relate to?
CALS-5426 Dropdown Chevron from Income Interval Covers up the text in the Edit Box
https://osi-cwds.atlassian.net/browse/CALS-5426
#### Brief feature description
- Dropdown view fix

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- 0%

### CODE REVIEW

**Comment by ratneshraval** [comment]
@vladimirPe I see the PR is created but no reviewers added. Also check other PRs and see how we are adding story link.

File: app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx [hunk 1]
```diff
@@ -58,8 +58,8 @@ export default class Employment extends React.Component {
               label='Occupation' placeholder=''
               type='text' onChange={(event) => this.onEmploymentChange('occupation', event.target.value)} />
 
-            <div className='col-md-3' >
-              <label>{'Personal Income'}</label>
+            <div className='col-md-2' >
+              <label>{'Income'}</label>
```
Comment (nasar-cambria) [inline app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: Is this approved by design ?

Comment (vladimirPe) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: yes it is

Comment (nasar-cambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: Nice. This is not your change but can you remove those ```{''}``` so just ```Income``` should be fine.

Comment (vladimirPe) [reply app/javascript/rfa_forms/rfa01a_edit_view/employmentCard.jsx]: Done

**Pull request merged**